.github/renovate.json

```diff
@@ -17,6 +17,12 @@
       "matchManagers": ["composer"],
       "groupName": "composer updates"
     },
+    {
+      "matchManagers": ["docker"],
+      "matchPackageNames": ["composer"],
```

Comment:
You should use `matchDepNames` instead to make sure it matches what you want (cf https://github.com/renovatebot/renovate/discussions/14885).
I had trouble with a python update once due to that.